bessctl/conf/samples/vxlan.bess

```diff
@@ -36,33 +36,34 @@ Container Bob  |    (regular Ethernet port)    |
 
 import os
 import socket
+import struct
 
-def aton(ip):
-    return socket.inet_aton(ip)
+def atoh(ip):
+    return struct.unpack("!L", socket.inet_aton(ip))[0]
 
-os.system('docker pull ubuntu > /dev/null')
-os.system('docker run -i -d --net=none --name=vxlan_test ubuntu /bin/bash > /dev/null')
+os.system('sudo docker pull ubuntu > /dev/null')
+os.system('sudo docker run -i -d --net=none --name=vxlan_test ubuntu /bin/bash > /dev/null')
 
 # Alice live outside the container, wanting to talk to Bob in the container
 v_bob = VPort(ifname='eth_bob', docker='vxlan_test', ip_addrs=['172.16.100.33/28'])
 v_alice = VPort(ifname='eth_alice', ip_addrs=['10.0.10.2/24'], mac_addr='02:0a:0b:0c:0d:0e')
 
 # Alice -> Bob
-PortInc(port=v_alice) -> bpf::BPF():1 -> VXLANDecap() -> PortOut(port=v_bob)
+PortInc(port=v_alice.name) -> bpf::BPF():1 -> VXLANDecap() -> PortOut(port=v_bob.name)
 
 # Bob -> Alice
-PortInc(port=v_bob) \
+PortInc(port=v_bob.name) \
     -> SetMetadata(attrs=
-            [{'name': 'tun_ip_src', 'size': 4, 'value': aton('10.0.10.1')},
-             {'name': 'tun_ip_dst', 'size': 4, 'value': aton('10.0.10.2')},
+            [{'name': 'tun_ip_src', 'size': 4, 'value': atoh('10.0.10.1')},
+             {'name': 'tun_ip_dst', 'size': 4, 'value': atoh('10.0.10.2')},
              {'name': 'tun_id', 'size': 4, 'value': 999}]) \
     -> VXLANEncap(dstport=4789) \
     -> IPEncap() \
     -> SetMetadata(attrs=
-            [{'name': 'ether_src', 'size': 6, 'value': '\x02\x01\x02\x03\x04\x05'},
-             {'name': 'ether_dst', 'size': 6, 'value': '\x02\x0a\x0b\x0c\x0d\x0e'}]) \
+            [{'name': 'ether_src', 'size': 6, 'value': 0x050403020102},
```

Comment:
The constant number 0x050403020102 is implicitly assuming a little-endian system. It would be nice to keep the code endian-neutral...

I think `value` used to accept both integer and string. If this is not feasible with PB, maybe we can have separate fields: `value_hex` (`value_str`?) and `value_int`.